packages/server/seeds/development/variants.js

```diff
@@ -9,118 +9,136 @@ exports.seed = async function (knex) {
   await knex('variants').insert([
     {
       color: 'red',
-      picture: '.jpg',
+      pictureUrl: 'packages/client/src/assets/jeans&shoes.jpg',
```

Comment:
@shpomp @zkwsk  I don't know if/how this will work 🤔 

When doing build, it will copy the assets from `src/assets` **if** they are imported by the code, and it will copy whatever is in the `public` folder. 
Since these URLs here might **not** be imported by the JS code, we have to make sure they're available in `public` and that the URLs used here match what it will be at build time, but I don't know at all how that will work at dev time...

I don't know if `create-react-app` actually accounts for this or if they assume that all assets are bundled with the code.